content/blog/intro-to-gamedev/index.md

```diff
@@ -0,0 +1,560 @@
+---
+title: Intro to game development with Typescript
+date: "2020-04-10T22:12:03.284Z"
+author: nacho
+description: "Creating a game is easier than ever with all the available tools. This article will hopefully ease you into game development, reviewing both general concepts and specific implementations with the help of robust technologies such as TypeScript and PixiJS!"
+tags:
+  - gamedev
+  - WebGL
+  - PixiJS
+  - TypeScript
+  - rendering
+---
+
+## Introduction
+
+If you are like me and are looking into getting out of Google's ecosystem, chances are that you switched (or are thinking on switching) from Chrome to another browser. In doing so, you will be getting rid of one of the cheekiest features in any browser: The [dinosaur game](https://elgoog.im/t-rex/).
+
+I thought we could take this chance to learn a little bit about web-based game development and create a small clone of the game.
+
+You can take a play the [finished game here](https://ignacioamigo.github.io/dinogame/bin/index.html) and [review the code here](https://github.com/IgnacioAmigo/dinogame). Keep in mind that the code is much more commented on this article since it's meant to be as explicit and educating as possible.
```

Comment:
you can `play` ?

```suggestion
You can play the [finished game here](https://ignacioamigo.github.io/dinogame/bin/index.html) and [review the code here](https://github.com/IgnacioAmigo/dinogame). Keep in mind that the code is much more commented on this article since it's meant to be as explicit and educating as possible.
```

Reply:
love this github feature

Reply:
ohh verdad, made so many changes/edits that this kind of thing probably happened more than once